### PR TITLE
multi-select functioning

### PR DESCRIPTION
**Pull request by allyr13**
Multi-select functioning properly - delete and move are working accurately

I would like to create a new task that implements a 'deselect' option. Currently, the reason I cannot implement it easily, is once selected, the action listener on the nodes is no longer active. So there is a work around there that makes me want to put it into a separate task. 

But the functionality of Ctrl+Click multi-select the nodes is working

### CODE REVIEW

File: network/src/components/AppWrapper.js [hunk 1]
```diff
@@ -590,24 +612,45 @@ export const AppWrapper = ({children}) => {
...
+    network.on('click', (event) => {
+      if (!event.nodes.length) {
+        handleCanvasClick(event);
+      }
+    });
+  };
+  
+  // only grabs the entity nodes (Person/Org)
+  const entitySelection = selectedNodes.filter(nodeId => {
+    const node = nodes.find(n => n.id === nodeId).nodeInfo;
```
Comment (simon-dunk) [inline network/src/components/AppWrapper.js]: I get an error on this line when trying to delete 1 or more nodes. 
image

**Review by simon-dunk — CHANGES_REQUESTED** [review]

File: network/src/components/AppWrapper.js (hunk 1 above)
Comment (allyr13) [reply network/src/components/AppWrapper.js]: Should be working now

**Comment by simon-dunk** [comment]
@allyr13 Error encountered when you 

*select a node* ->
 *delete the node* -> 
*without clicking anywhere else first* -> 
*control (or command) click another node* ->
*error thrown: "Node with id "<uuid>" not found"*. 

This would be caused because the program is trying to add a deleted node to the list of multiple nodes selected. I assume adding some sort of current node selected list reset when the delete function is called should fix this issue. I.e. when nodes are deleted, they are no longer part of the selected node set.

**Comment by allyr13** [comment]
@simon-dunk I believe it should be fully working now

File: network/src/components/AppWrapper.js [hunk 2]
```diff
@@ -646,8 +724,13 @@ export const AppWrapper = ({children}) => {
...
               <button class='addPerson' onClick={() => {
-                  setPersonDialog(true);      // Open the "Add Person" dialog
-                  resetCanvasContextMenu();   // Reset the canvas context menu
+                  setPersonDialog(true);     
+                  resetCanvasContextMenu();  
+                  }} 
+                  style={{ padding: '10px 20px', 
+                    margin: '5px', 
+                    backgroundColor: 'green', 
+                    color: 'white'
```
Comment (simon-dunk) [inline network/src/components/AppWrapper.js]: In 0.0.2, this does not exist. These changes were made in PR #27. Before this branch can be merged, these changes need to be brought into parody. In your local branch, run `git pull` then `git merge 0.0.2`. You may need to resolve some conflicts after that. Once those changes are resolved, commit & push your local branch.

**Review by simon-dunk — CHANGES_REQUESTED** [review]

**Review by simon-dunk — APPROVED** [review]

File: network/src/components/AppWrapper.js (hunk 2 above)
Comment (allyr13) [reply network/src/components/AppWrapper.js]: Updated

**Pull request merged**